build/Makefile

```diff
@@ -346,19 +346,32 @@ push-build-image:
 	docker tag $(build_tag) $(build_remote_tag)
 	docker push $(build_remote_tag)
 
-# port forward the agones controller.
-# useful for pprof and stats viewing, etc
-controller-portforward: PORT ?= 8080
-controller-portforward:
-	docker run --rm -it $(common_mounts) $(DOCKER_RUN_ARGS) \
-		-e "KUBECONFIG=/root/.kube/$(kubeconfig_file)"  -p $(PORT):$(PORT) $(build_tag) \
-			kubectl port-forward deployments/agones-controller -n agones-system $(PORT)
-
 # start pprof with a web ui
 pprof-web:
 	docker run --rm -it --network=host $(common_mounts) $(DOCKER_RUN_ARGS) $(build_tag) \
 		go tool pprof -http :6061 http://localhost:6060
 
+# setup prometheus in the current cluster by default Persistent Volume Claims are requested.
+setup-prometheus: PVC ?= true
```

Comment:
Random idea:
Should the implementation of setup-prometheus + setup-grafana be .sh files, that we can point Agones users to (and then the Makefile reuses) - reason being, then people who want a quick up and running prom+grafana don't have to invest in the entire makefile,  but instead could run the .sh files instead?

We could still keep the targets for Agones developers, but use the .sh scripts instead for the details.

WDYT?

Reply:
So I think I'm going to improve the docs to give installation instructions with helm & kubectl instead and give hint that you could also use our make targets.